compiler/rustc_hir/src/hir.rs

```diff
@@ -2890,7 +2890,7 @@ pub struct ForeignItemRef<'hir> {
 pub struct ForeignItem<'hir> {
     pub ident: Ident,
     pub kind: ForeignItemKind<'hir>,
-    pub def_id: LocalDefId,
+    pub def_id: HirOwner,
```

Comment:
I think `owner` would be a better name now for this field here and elsewhere.

Reply:
There is already a struct Owner here https://github.com/rust-lang/rust/blob/fbf1b1a7193cda17008ab590e06ad28d9924023b/compiler/rustc_middle/src/hir/mod.rs#L34-L37